test/image/compare_pixels_test.js

```diff
@@ -107,16 +105,13 @@ for(var i = 0; i < allMockList.length; i++) {
     var mockName = allMockList[i];
 
     // skip blacklist
-    if([
-        'mapbox_density0-legend',
-        'mapbox_osm-style'
-    ].indexOf(mockName) !== -1) {
-        continue;
-    }
+    if(blacklist.indexOf(mockName) !== -1) continue;
 
     var isMapbox = mockName.substr(0, 7) === 'mapbox_';
     var isOtherFlaky = [
         // list flaky mocks other than mapbox:
+        'map_density0-legend',
```

Comment:
@archmoj why are these here rather than in the blacklist above?

Reply:
In the blacklist we didn't produce the baselines in the previous step. 
https://github.com/plotly/plotly.js/blob/c8f57028c0cfff778b472f5449a520670472933b/test/image/make_baseline.py#L97

So we basically skip them in the comparison test. 
For flaky ones we create the baselines and compare them with some tolerance.